packages/features/bookings/lib/handleNewBooking/test/fresh-booking.test.ts

```diff
@@ -1374,7 +1439,7 @@ describe("handleNewBooking", () => {
             status: BookingStatus.PENDING,
           });
 
-          expectWorkflowToBeTriggered();
+          expectWorkflowToBeNotTriggered({ organizer, emails });
```

Comment:
status is pending because event type requires confirmation, workflow is triggered when booking is confirmed